Rater.Business/Services/SpaceService.cs

```diff
@@ -12,25 +16,48 @@ namespace Rater.Business.Services
     public class SpaceService : ISpaceService
     {
 
-        private readonly ISpaceRepository _repo;
-        public SpaceService(ISpaceRepository repo)
+        private readonly ISpaceRepository _spaceRepo;
+        private readonly IUserRepository _userRepo;
+        private readonly IMapper _mapper;
+        public SpaceService(ISpaceRepository spaceRepo,IUserRepository userRepo,IMapper mapper)
         {
-            _repo = repo;
+            _spaceRepo = spaceRepo;
```

Comment:
Using other repos directly in SpaceService is not a good practice. Each service should only use it's own repo class.